cbfeeds/feed.py

```diff
@@ -87,46 +95,67 @@ def validate(self, pedantic=False, serialized_data=None):
 class CbFeedInfo(object):
     def __init__(self, **kwargs):
         # these fields are required in every feed descriptor
-        self.required = ["name", "display_name", "version",
```

Comment:
OK @scottlundgren says that we never referenced the "version" parameter. We can take it out of the "required" list as long as the code doesn't break if it's passed in anyway (since previous versions implied/stated that it was required)

Note that "version" is still reflected in the README, so add a patch to remove those references from that document as part of this branch as well.